### ISSUE TITLE
Align default JMX endpoint exposure with default web endpoint exposure

### ISSUE THREAD

**Issue by wilkinsona** [spring-projects/spring-boot#32005]
We currently expose more endpoints by default over JMX than we do over the web. For consistency, we've decided to change the default exposure over JMX to align it with the default exposure over the web.

**Comment by kaleev** [comment]
@mbhave, could you please also fix `defaultValue` for `management.endpoints.jmx.exposure.include` in `additional-spring-configuration-metadata.json`?

**Comment by philwebb** [comment]
@kaleev I've opened https://github.com/spring-projects/spring-boot/issues/33072

**Comment by gmuth** [comment]
I understand the idea, however considering that JMX by default is only exposed locally and Web is exposed to remote connections it should be okay to have different default settings IMHO.

This change has an immediate impact on IDEs integrating with spring boot actuators via JMX. Intellij IDEA notifies on application start up about missing endpoint data. Developers now have to explicitly enable those endpoints:

```
# file: application.properties
# fix IntelliJ IDEA actuator tool issues:
# "Failed to receive 'mappings' endpoint data" 
# "Failed to retrieve 'beans' endpoint data"
management.endpoints.jmx.exposure.include=mappings,beans
```
 
This should at least be clarified in the reference documentation or/and in the "Migrate to SpringBoot 3.0" documentation.

**Comment by wilkinsona** [comment]
@gmuth It's already [mentioned in the release notes](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0.0-M5-Release-Notes#jmx-endpoint-exposure) in the "Upgrading from Spring Boot 2.x" section.

**Comment by kaleev** [comment]
@gmuth it has been already fixed in Intellij IDEA: https://youtrack.jetbrains.com/issue/IDEA-301274
The fix will be included in the upcoming release 2022.3.
